### PR TITLE
chore: fix error in show parent

### LINKED ISSUE #29489: Show the path in selected items
### Parent Issue

https://github.com/dotCMS/core/issues/28493

### User Story

When a user selects an item, the path of each selected item should show the proper path.

image

### Acceptance Criteria

Show the path in selected items

### Proposed Objective

User Experience

### Proposed Priority

Priority 4 - Trivial

### External Links... Slack Conversations, Support Tickets, Figma Designs, etc.

_No response_

### Assumptions & Initiation Needs

_No response_

### Quality Assurance Notes & Workarounds

_No response_

### Sub-Tasks & Estimates

_No response_

### PR DESCRIPTION
**Pull request by nicobytes**
### Parent Issue

https://github.com/dotCMS/core/issues/29489

### Proposed Changes
* Fix bug in map to show path
* Add query param to get ParentList attr

### Checklist
- [x] Tests
- [x] Translations
- [x] Security Implications Contemplated (add notes if applicable)

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
Screenshot 2024-08-08 at 10 12 38 AM |  Screenshot 2024-08-08 at 10 13 00 AM

This PR fixes: #29489

This PR fixes: #29489

### CODE REVIEW

**Comment by nicobytes** [comment]
https://github.com/user-attachments/assets/4acbcde4-efec-4dc3-8ec3-e55e8e73acbc

**Comment by fmontes** [comment]
Out of topic but

CleanShot 2024-08-19 at 08 55 07@2x

**Comment by nicobytes** [comment]
> Out of topic but
> 
> CleanShot 2024-08-19 at 08 55 07@2x

Fixed!

Screenshot 2024-08-19 at 1 33 12 PM

File: core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-sidebar/dot-category-field-sidebar.component.scss [hunk 1]
```diff
@@ -69,7 +68,7 @@
 
 .category-field__selected-categories-list {
     overflow-y: auto;
-    scrollbar-gutter: stable;
+    scrollbar-gutter: auto;
```
Comment (KevinDavilaDotCMS) [inline core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-category-field-sidebar/dot-category-field-sidebar.component.scss:71]: TIL

**Review by KevinDavilaDotCMS — APPROVED** [review]

**Review by zJaaal — APPROVED** [review]

**Pull request merged**